### PR TITLE
Add detect rhel version to testkomodo script

### LINKED ISSUE #663: Subscript testing uses hardcoded paths binaries and libraries
Subscript testing uses hardcoded paths binaries and libraries.
This is working fine when run in the `RHEL7` environment, but breaks pretty fast when using the `RHEL8` environment.

[ci/testkomodo.sh](https://github.com/equinor/subscript/blob/011afce91ae8df3dddb97dd4f2e59b9542fe2583/ci/testkomodo.sh#L9)
```
pytest -n auto --flow-simulator="/project/res/x86_64_RH_7/bin/flowdaily" --eclipse-simulator="runeclipse"
```

[docs/contributing.rst](https://github.com/equinor/subscript/blob/011afce91ae8df3dddb97dd4f2e59b9542fe2583/docs/contributing.rst#L61)
```
pytest -n auto --flow-simulator="/project/res/x86_64_RH_7/bin/flowdaily" --eclipse-simulator="runeclipse"
```

[tests/testdata_runrms/runrms.yml](https://github.com/equinor/subscript/blob/011afce91ae8df3dddb97dd4f2e59b9542fe2583/tests/testdata_runrms/runrms.yml#L41)
```
- /prog/res/roxapi/x86_64_RH_7/matrix/12.1/lib/python3.8/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/12.1_test/lib/python3.8/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/11.1/lib/python3.6/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/11.1_test/lib/python3.6/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/11.1/plugins
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/havana
pythonpathtest:
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0_test/lib/python3.6/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/plugins
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/havana
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0_test/lib/python3.6/site-packages
pluginspath:
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/plugins
tcltkpath: /prog/roxar/rms/versions/12.0.1/lib/LINUX_64/tcl8.6
- /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
```

[tests/test_ri_wellmod.py](https://github.com/equinor/subscript/blob/011afce91ae8df3dddb97dd4f2e59b9542fe2583/tests/test_ri_wellmod.py#L12)
```
RI_DEV = "/project/res/x86_64_RH_7/share/resinsight/jenkins_dev/ResInsight"
```

### PR DESCRIPTION
**Pull request by andreas-el**
Resolves #663

### CODE REVIEW

File: tests/testdata_runrms/runrms.yml [hunk 1]
```diff
@@ -17,49 +17,49 @@ rms:
...
     tcltkpath: /prog/roxar/rms/versions/12.0.1/lib/LINUX_64/tcl8.6
 
 # list of nonstandard installs (experimental)
 rms_nonstandard:
   12.0.0:
     exe: /prog/roxar/site/RMS12/rms/rms
     pythonpath:
-      - /prog/res/roxapi/x86_64_RH_7/matrix/12.0/lib/python3.6/site-packages
+      - /prog/res/roxapi/<PLATFORM>/matrix/12.0/lib/python3.6/site-packages
     tcltkpath: /prog/roxar/site/RMS12/rms/versions/12.0.0/lib/LINUX_64/tcl8.6
```
Comment (andreas-el) [inline tests/testdata_runrms/runrms.yml:65]: These changes should be reverted @larsevj ?
You mentioned no rhel8 rms specific version available, and the current rhel7 version seems to be ok?

File: tests/test_ri_wellmod.py [hunk 2]
```diff
@@ -3,13 +3,16 @@
...
+from subscript import detect_os
 from subscript.ri_wellmod import ri_wellmod
 
 pytestmark = pytest.mark.xfail()
 
 SCRIPTNAME = "ri_wellmod"
 DATAPATH = Path(__file__).parent / "testdata_ri_wellmod"
-RI_DEV = "/project/res/x86_64_RH_7/share/resinsight/jenkins_dev/ResInsight"
+
+RI_DEV = "/project/res/<PLATFORM>/share/resinsight/jenkins_dev/ResInsight"
```
Comment (andreas-el) [inline tests/test_ri_wellmod.py:14]: This folder does not exist in neither RHEL7 nor RHEL8 setups

Comment (andreas-el) [reply tests/test_ri_wellmod.py]: Reverted since not important atm.

**Review by berland — APPROVED** [review]
lgtm

**Pull request merged**